.github/issue_template.md

```diff
@@ -0,0 +1,44 @@
+---
+labels: web-ui
+---
+
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Expected Behavior
+<!--- If you're looking for help, please see https://sensuapp.org/support for resources --->
+<!--- If you're describing a bug, tell us what should happen -->
+<!--- If you're suggesting a change/improvement, tell us how it should work -->
+
+## Current Behavior
+<!--- If describing a bug, tell us what happens instead of the expected behavior -->
+<!--- If suggesting a change/improvement, explain the difference from current behavior -->
+
+## Possible Solution
+<!--- Not obligatory, but suggest a fix/reason for the bug, -->
+<!--- or ideas as to the implementation of the addition or change -->
+
+## Steps to Reproduce (for bugs)
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code or configuration to reproduce, if relevant -->
+1.
+2.
+3.
+4.
+
+## Context
+<!--- How has this issue affected you? What are you trying to accomplish? -->
+<!--- Providing context (e.g. links to configuration settings, stack strace or log data) helps us come up with a solution that is most useful in the real world -->
+
+## Customers
+<!--- A list of the customers affected or requesting, with links for context -->
+1.
+2.
+3.
+4.
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in -->
+* Sensu version used (sensuctl, sensu-backend, and/or sensu-agent):
+* Installation method (packages, binaries, docker etc.):
+* Operating System and version (e.g. Ubuntu 14.04):
```

Comment:
In the case of the web app maybe we should ask for the browser as well as the sensu version?

Reply:
Adding browser and version are great additions